types/Cargo.toml

```
@@ -18,7 +18,7 @@ blake2 = { version = "0.9.0", default-features = false }
 datasize = { version = "0.2.0", default-features = false }
 failure = { version = "0.1.8", default-features = false, features = ["failure_derive"] }
 hex_fmt = "0.3.0"
-lazy_static = { version = "1.4.0", optional = true }
+once_cell = "1.5.2"
```

We should only use `once_cell` when we have our `std` feature enabled, since according to its docs `once_cell` isn't compatible with a `no_std` environment (our default for the `casper_types` crate).
```suggestion
once_cell = { version = "1.5.2", optional = true }
```

---

Added suggestion